cmd/lifecycle/main.go

```diff
@@ -76,6 +82,90 @@ func subcommand(platform Platform) {
 	}
 }
 
+// handlers
```

Comment:
These seem fine to live in the `main` package, because they are very hard to unit test and will be covered by acceptance.